roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -362,6 +372,17 @@ spec:
             items:
               - key: receptor_conf
                 path: receptor.conf
+        - name: "{{ ansible_operator_meta.name }}-projects"
+{% if projects_persistence|bool %}
+          persistentVolumeClaim:
+{% if projects_existing_claim %}
+            claimName: {{ projects_existing_claim }}
+{% else %}
+            claimName: '{{ ansible_operator_meta.name }}-projects-claim'
+{% endif %}
+{% else %}
+          emptyDir: {}
+{% endif %}
```

Comment:
I dont think this will work?

Reply:
since web and task deployment can not share the same emptyDir

Reply:
ok so mount only if project_persistence is set, that makes sense